tmcprototype/DishMaster/DishMaster/DishMaster.py

```diff
@@ -59,6 +60,10 @@ def point(self):
         if((self._achieved_pointing[1] != self._desired_pointing[1]) |
            (self._achieved_pointing[2] != self._desired_pointing[2])):
             try:
+                self._azimuth_difference = self._desired_pointing[1] - self._achieved_pointing[1]
+                print("azimuth difference is: ", self._azimuth_difference)
+                self._elevation_difference = self._desired_pointing[2] - self._achieved_pointing[2]
+                print("elevation difference is: ", (self._elevation_difference))
                 self.change_azimuth_thread = threading.Thread(None, self.azimuth, 'DishMaster')
                 self.change_elevation_thread = threading.Thread(None, self.elevation, 'DishMaster')
                 self.change_azimuth_thread.start()
```

Comment:
Is it also not required to start self.change_elevation_thread?

Reply:
Yes. It is required and self.change_elevation_thread start statement is already there at line 70 in DishMaster.py